source/libs/features.tsx

```diff
@@ -98,24 +91,19 @@ const globalReady: Promise<GlobalOptions> = new Promise(async resolve => {
 	document.documentElement.classList.add('refined-github');
 
 	// Options defaults
-	const options: GlobalOptions = {
-		disabledFeatures: '',
-		customCSS: '',
-		logging: false,
-		...await new OptionsSync().getAll()
-	};
-
-	if (options.customCSS.trim().length > 0) {
+	const options = await new OptionsSync().getAll();
+
+	if ((options.customCSS as string).trim().length > 0) {
```

Comment:
```suggestion
	if (options.customCSS.trim().length > 0) {
```

Reply:
`as string` has been introduced to avoid that error. If I drop `as string` I get 

```
TS2339: Property 'trim' does not exist on type 'string | number | boolean'.
  Property 'trim' does not exist on type 'number'.
```

Reply:
It's fine if you cast as I mentioned above, `const options = (await new OptionsSync().getAll()) as Options;`.